docker/wrapper.py

```diff
@@ -12,37 +12,131 @@
 import gzip
 from bd2k.util.exceptions import require
 from toil.lib.bioio import addLoggingOptions, setLoggingFromOptions
+import time
+import virtualenv
+import re
+from itertools import chain
+
+from shutil import copyfile
+#import ConfigParser
```

Comment:
Remove line and optimize imports (Pycharm `ctrl + option + o`)

Reply:
done